src/fundus/publishers/us/fox_news.py

```diff
@@ -14,7 +14,7 @@
 
 class FoxNewsParser(ParserProxy):
     class V1(BaseParser):
-        _paragraph_selector = CSSSelector(".article-body > p")
+        _paragraph_selector = XPath("//div[@class='article-body'] / p[text()]")
 
         @attribute
         def body(self) -> ArticleBody:
```

Comment:
I think we could add a summary/subtitle selector for FoxNews as well

Reply:
Good point, i will add this as wel.